mkdocs.yml

```
@@ -1,6 +1,7 @@
 ---
 site_name: Hera
 site_author: Flaviu Vadan, Sambhav Kothari, Elliot Gunton
+site_url: https://hera-workflows.readthedocs.io/en/latest/
```

This should be 
```suggestion
site_url: https://hera.readthedocs.io/en/latest/
```

As `hera-workflows` is for the old package name (which will eventually be removed)

---

Aha great, thank you